src/cerebral.d.ts

```diff
@@ -0,0 +1,107 @@
+﻿declare module "cerebral" {
```

Comment:
Not sure that's gonna work. We need to bundle them separately from cerebral in the @types repo. There will be different versions of them.